tests/neg-custom-args/captures/heal-tparam-cs.check

```diff
@@ -14,17 +15,15 @@
 -- [E007] Type Mismatch Error: tests/neg-custom-args/captures/heal-tparam-cs.scala:15:13 -------------------------------
 15 |    localCap { c =>  // error
    |    ^
-   |    Found:    (x$0: Capp^'s4) ->'s5 () ->{x$0} Unit
-   |    Required: (c: Capp^) -> () => Unit
+   |Found:    (x$0: Capp^'s4) ->'s5 () ->{x$0} Unit
+   |Required: (c: Capp^) -> () => Unit
    |
-   |    Note that capability x$0 is not included in capture set {cap}.
+   |Note that capability x$0 is not included in capture set {cap}
+   |because cap, which is existentially bound in () =>² Unit, cannot subsume x$0 since that capability is not a SharedCapability.
```

Comment:
the number 2 in `() =>² Unit` seems to be out of nowhere compared to the preceding context.